### ISSUE TITLE
Live bets should be publicly acessible

### ISSUE THREAD

**Issue by davidsharff** [davidsharff/gonfalon-sports-book#11]
For the initial rollout, I think we should just have a seperate view showing a list of all the active bets.

We could do other cool things later like show current bet details as part of the prop record/description.

**Comment by NateMeyvis** [comment]
Yep. That seems right. Also down the line, we'll want to let the market decide the lines rather than dictating them from on high.

**Comment by davidsharff** [comment]
> we'll want to let the market decide the lines rather than dictating them from on high.

Good reminder. In deference to our future selves, we should probably use selector functions to access any data that will end up being dynamic (on the server and client). For now, they'll be simple pass through functions, but later it will be easy to inject the calculations.
